initfiles/componentfiles/configxml/thor.xsd.in

```diff
@@ -462,14 +462,14 @@
           </xs:appinfo>
         </xs:annotation>
       </xs:attribute>
-      <xs:attribute name="masterport" type="xs:nonNegativeInteger" default="20000">
+      <xs:attribute name="masterport" type="xs:nonNegativeInteger" default="20000" use="optional">
```

Comment:
I am not entirely sure why I changes this.. possibly unintentionally.
Will check and probably revert.